css/main.css

```diff
@@ -1,3 +1,203 @@
-/* Add your custom CSS here */
 
```

Comment:
The browser adds a default 8px margin on the body which is why there is a white edge around your page. 
Add this to remove it:
```
body {
   margin: 0;
}
```